prisum/static/vend/bootstrap-3.3.5/bootstrap.css

```diff
@@ -6293,9 +6295,9 @@ button.close {
 .carousel-inner > .item {
   position: relative;
   display: none;
-  -webkit-transition: .6s ease-in-out left;
-       -o-transition: .6s ease-in-out left;
-          transition: .6s ease-in-out left;
+  -webkit-transition: 5s ease-in-out left;
+       -o-transition: 5s ease-in-out left;
+          transition: 5s ease-in-out left;
```

Comment:
You can remove the stuff that used to be for the carousel, as in here and in the .js file since it's not needed. In general we would want to apply these to your sponsors.css file instead of modifying the bootstrap.css or bootstrap.js files. You would just add another class name in the html elements and give that class the extra style attributes in your sponsors.css file.